### ISSUE TITLE
Source for Texts from Manuscripts

### ISSUE THREAD

**Issue by jedwardwalters** [srophe/syriac-corpus-app#71]
Some of the texts in the corpus come from manuscripts, not from published editions. The system we have for displaying the "Source" does not grab the manuscript information. See for example:
http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/26 

This may be something I need to fix in the TEI header. Just let me know.

**Comment by jedwardwalters** [comment]
Also, the source isn't displaying for some other texts that do come from a printed edition, so this issue is worth looking into more broadly: http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/26

Again, this may just be something I did wrong on the bibl element.

**Comment by wsalesky** [comment]
@jedwardwalters Can you take one example and print how you expect it to appear in the issue, so I can track down what is going wrong?

**Comment by wsalesky** [comment]
@jedwardwalters These are coded with msDesc, not biblStruct. I will need to add that to the output logic. How would you like them to display? Using http://wwwb.library.vanderbilt.edu/exist/apps/syriac-corpus/26 as an example.

**Comment by jedwardwalters** [comment]
I think just whatever shows up in the msName tag would be fine to display. So, in the case of /26, it would be: Vatican syr. 117, fol. 412b-32a

Note: most of them won't have multiple msName tags. If that causes a problem here, we could probably just delete the second one.

I included the folio pages in msName because, at least from my poking around, I didn't see anything like a folio/page range that would work in msDesc. But I may have just missed it.

**Comment by wsalesky** [comment]
Do you want to display all msName tags? Separated by a period? Or just the first?

**Comment by jedwardwalters** [comment]
I suppose it would be good to include all of them if there are multiples. Would it be possible to list each one separately as a source? So, like this:
Source: Vatican syr. 117, fol. 412b-32a
Source: Mardin 136

If that is too complicated, I think maybe we should just list the first one to avoid confusion. If we were encoding a critical edition made from multiple mss, we'd probably do this whole section differently anyway. Which reminds me: I should try to find an example of a critical edition so we can work on that.

**Comment by wsalesky** [comment]
That is fine. Changes will be up shortly.

**Comment by wsalesky** [comment]
That should take care of the problem.

**Comment by jedwardwalters** [comment]
@wsalesky So sorry to re-open this. Your solution did fix it, but I should have dug deeper before I responded. On other ones from manuscripts, I fleshed out the msDesc better, and think this is a better use of the TEI. See the example here:
https://github.com/srophe/syriac-corpus/blob/master/data/tei/32.xml#L75

I think it would be great if we could do something like this:
Source: [settlement], [repository] [idno]

So, for the example given above, it would look like this:
Oxford, British Library Oriental 5463

(Though, now that I think about it, that should be "London" and not "Oxford"! I'll change that)

If it would help, since there are other appearances of the idno element, I could specify the type of that idno as a shelfmark.

Sorry to make more work on this.

**Comment by wsalesky** [comment]
Sounds good!

**Comment by jedwardwalters** [comment]
Let me know if it would be helpful to have the type marked as shelfmark on that idno element. It might be good to do that anyway, but it's especially worth it if it makes your job easier.

**Comment by wsalesky** [comment]
Yes, that would be useful.

**Comment by jedwardwalters** [comment]
On it.

**Comment by jedwardwalters** [comment]
Okay, I've updated all the texts in question. I fixed the ones that were using msName so that they now follow the Settlement, Repository, idno structure and added "shelfmark" as a type.

One more tiny change to what I suggested above: let's go ahead and add a comma between Repository and idno in the display. So:
Source: [Settlement], [Repository], [idno type="shelfmark"]
Source: London, British Library, Oriental 5463


None of the examples thus far have specific folio ranges. But I think I found the right way to include that if we do come across an example. But we'll just have to cross that bridge when we get there!
